src/Fixer/Basic/BracesFixer.php

```diff
@@ -30,17 +30,26 @@
  */
 final class BracesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    const NEXT_LINE = 'next';
+
+    const SAME_LINE = 'same';
+
+    private $supportedBracePositions = array(self::NEXT_LINE, self::SAME_LINE);
+
     /**
      * @var array
      */
     private $configuration;
 
     private static $defaultConfiguration = array(
         'allow_single_line_closure' => false,
+        'classy_constructs' => self::NEXT_LINE,
```

Comment:
As I wrote before, this does not cover `if` statements. See the documentation: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2459/files#diff-0b7fa2dc2594eae7a18836044e3287d8R172

Though it does affect non-anonymous functions since they are handled by the same code as methods. I cannot come up with a better hypernym, though.